chef_master/source/chef_license_accept.rst

```diff
@@ -0,0 +1,277 @@
+The same license applies to all products, but each product must have its own license acceptance.
+``chef --chef-license accept`` will accept the license for Chef Workstation, Chef Infra, Chef InSpec, and Push Jobs Client.
+
+Chef Infra
+-----------------------------------------------------
+In addition to the above methods, users can specify ``chef_license = 'accept'`` in their Chef config.
+On a workstation, this can be specified in ``~/.chef/config.rb`` or ``~/.chef/knife.rb``, and on a node, it can be specified in ``/etc/chef/client.rb``.
+This method of license acceptance is backwards-compatible to non-EULA versions of Chef Infra.
+
+Habitat
+-----------------------------------------------------
+The Habitat license can be accepted by setting either environment variables ``HAB_LICENSE=accept`` or ``HAB_LICENSE=accept-no-persist``, or by executing ``hab license accept``.
+If neither of these are set, the first ``hab`` invocation will ask the user to accept the license through an interactive prompt.
+
+Note: This does not apply to packages installed using Habitat.
+In those cases, a Habitat configuration is set per package.
+Documentation for this is coming soon.
+
+Server Products
+=====================================================
+* Chef Server, which also contains
+    * Push Jobs Server
+* Chef Automate
+* Supermarket
+
+Server products are typically installed and managed by some kind of supervisor instead of a user on a workstation.
+Because of this, there is no interactive option for accepting the license.
+Instead, the license is accepted during the ``reconfigure`` command or ``upgrade`` command for the Omnibus ctl command.
+For example:
+
+* ``chef-server-ctl reconfigure --chef-license=accept``
+* ``CHEF_LICENSE=accept-no-persist supermarket-ctl reconfigure``
+
+In addition, the Chef license can be accepted via the omnibus configuration file.
+Specify ``chef_license 'accept'`` in the ``chef-server.rb`` or ``supermarket.rb`` configuration.
+
+Chef Automate
+-----------------------------------------------------
+Automate has its own reconfigure tool, ``automate-ctl``.
+This tool walks users through the install and setup of Automate.
+The Chef license is accepted after that in the browser.
+Please follow the in-product prompts.
+
+Remote Management Products
+=====================================================
+* Test Kitchen
+* ``knife bootstrap`` in Chef Infra
+* Vagrant
+* Chef Terraform Provisioner
+* Habitat Terraform Provisioner
+* Packer
+
+These products install or manage Chef on a remote instance.
+If a user has accepted the appropriate product license locally, it will be automatically transferred to the remote instance.
+This means that if a user has accepted the Chef Infra license locally and converges a Test Kitchen instance with the Chef provisioner, it will succeed by copying the acceptance to the remote instance.
```

Comment:
```suggestion
This means that if a user has accepted the Chef Infra Client license locally and converges a Test Kitchen instance with the Chef provisioner, it will succeed by copying the acceptance to the remote instance.
```